app/controllers/hosts_controller.rb

```diff
@@ -540,7 +541,7 @@ def process_hostgroup
 
   def process_taxonomy
     return head(:not_found) unless @location || @organization
-    @host = Host.new(params[:host].except(:interfaces_attributes))
```

Comment:
I think you want to keep this, otherwise we get `ActiveModel::MassAssignmentSecurity::Error (Can't mass-assign protected attributes: 0, 1, new_interfaces)` in the logs because of the mass assignment (even though it's fixed by the filter).